src/tests/unit/tests/background/injector-controller.test.ts

```diff
@@ -177,7 +177,7 @@ class InjectorControllerValidator {
 
     public setupTimeoutHandler(times: number): InjectorControllerValidator {
         this.mockWindowUtils
-            .setup(x => x.setTimeout(It.is(isFunction), It.isAnyNumber()))
+            .setup(x => x.setTimeout(isFunction, It.isAnyNumber()))
```

Comment:
When I look at a test using this new helper, it kind of looks like the test has a bug because the normal usage of these mocks is usually of the form "It.isSomething". Maybe it would be more clear to use a name like "itIsFunction", even though this would be an exception to our normal conventions about starting functions with verbs?

Reply:
Agreed

Reply:
@dbjorge Updated